src/expint.jl

```diff
@@ -297,29 +297,57 @@ end
 # series about origin, general ν
 # https://functions.wolfram.com/GammaBetaErf/ExpIntegralE/06/01/04/01/01/0003/
 function En_expand_origin_general(ν::Number, z::Number)
-    gammaterm = En_safe_gamma_term(ν, z)
-    frac = one(real(z))
-    sumterm = frac / (1 - ν)
+    # gammaterm = En_safe_gamma_term(ν, z)
+    gammaterm = gamma(1-ν)*z^(ν-1)
+    frac = one(z)
+    blowup  = abs(1 - ν) < 0.5 ? frac / (1 - ν) : zero(z)
+    sumterm = abs(1 - ν) < 0.5 ? zero(z) : frac / (1 - ν)
     k, maxiter = 1, 100
-    ϵ = 10*eps(real(sumterm))
+
     while k < maxiter
         frac *= -z / k
         prev = sumterm
-        sumterm += frac / (k + 1 - ν)
-        if abs(sumterm - prev) < ϵ
-            break
+        if abs(k + 1 - ν) < 0.5
+            blowup += frac / (k + 1 - ν)
+        else
+            sumterm += frac / (k + 1 - ν)
+            if abs(sumterm - prev) < 10*eps(real(prev))
```

Comment:
Shouldn't this be `abs(prev)`?  In case the real part happens to be converging to zero?  Or maybe that never happens where the Taylor series is used?